lib/studies/reply/inbox.dart

```diff
@@ -8,27 +8,35 @@ class InboxPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final isDesktop = isDisplayDesktop(context);
+    final isTablet = isDisplaySmallDesktop(context);
 
-    return SafeArea(
-      child: Row(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Expanded(
-            child: ListView.separated(
-              padding: EdgeInsetsDirectional.only(
-                start: isDesktop ? 120 : 6,
-                end: isDesktop ? 120 : 6,
-                top: isDesktop ? 28 : 6,
-              ),
-              itemCount: 2,
-              separatorBuilder: (context, index) => const SizedBox(height: 6),
-              itemBuilder: (context, index) {
-                return const MailPreviewCard();
-              },
+    return Navigator(
```

Comment:
Why do you need to introduce a new Navigator?

Reply:
This is to keep the bottom app bar or nav rail/drawer persistent across all screens. If not when pressing on an email, the incoming screen takes up the whole scaffold, instead of just the area above the bottom app bar or to the right of the navigation rail/drawer.